test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -847,6 +845,25 @@ define([
             },
         ];
 
+        Object.keys(ResponseData).forEach((respType) => {
```

Comment:
for each of the different backend response message types, ensure that all messages are distributed correctly